platform/admin/ui/pom.xml

```diff
@@ -66,6 +68,10 @@
             <type>tar.gz</type>
             <classifier>npm</classifier>
         </dependency>
+        <dependency>
```

Comment:
No I don't think so, actually I think `org.apache.karaf.webconsole.console` can be removed from several poms.